notebooks/en/chain_of_verification.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        verification_completion = await config.run("verification", params, options=inference_options)
</pre><p>I'm getting an error on this line:</p><p> </p><pre class="ql-syntax" spellcheck="false">UnwrapError: Called <code>Result.unwrap()</code> on an <code>Err</code> value: "Missing API key 'OPENAI_API_KEY' in environment"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>